.github/workflows/wdio-tests.yml

```
@@ -31,66 +51,48 @@ jobs:
-          cache: 'npm'
-          cache-dependency-path: |
-            package-lock.json
-      - name: Install dependencies
-        run: npm install
       - name: WDIO Tests
         env:
-          PGHOST: localhost
-          PGUSER: postgres
-          PGPASSWORD: postgres
-          PGDATABASE: ci_db_test
           PGPORT: ${{ job.services.postgres.ports[5432] }}
```

actionlint (1.7.12)

[error] property access of object must be type of string but got "number"

(expression)